draft-ietf-quic-transport.md

```diff
@@ -1716,11 +1716,12 @@ Note:
   control, which are not expected to be relevant for a closed connection.
   Retransmitting the final packet requires less state.
 
-After receiving a closing frame, endpoints enter a draining period.  An endpoint
+After receiving a closing frame, endpoints enter a draining state.  An endpoint
 that receives a closing frame MAY send a single packet containing a
-CONNECTION_CLOSE frame before entering a draining period, but MUST NOT send
-further packets, which could result in a constant exchange of closing frames
-until the closing period on either peer ended.
+closing frame before entering a draining state, using the NO_ERROR code if no
```

Comment:
You are replacing the "CONNECTION_CLOSE frame" reference by "closing frame". Actually, I think it is more logical to use the "Connection Close" frame there, since "Application Close" is suppose to only be done at the application's initiative.

Reply:
oops, yeah, that's obvious now, thanks

Reply:
I guess the correct text would be "send a CONNECTION CLOSE unless the application has already requested closing the connection." But that gets kind of long, and we can also let the developers decide. In any case, a peer in the closing state has to expect either due to the possibility of simultaneous close, so I guess it does not matter for interop.